akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -342,31 +343,32 @@ private[akka] class Shard(
 
 private[akka] object RememberEntityStarter {
   def props(
+    region:    ActorRef,
     typeName:  String,
     shardId:   ShardRegion.ShardId,
     ids:       Set[ShardRegion.EntityId],
     settings:  ClusterShardingSettings,
     requestor: ActorRef) =
-    Props(new RememberEntityStarter(typeName, shardId, ids, settings, requestor))
+    Props(new RememberEntityStarter(region, typeName, shardId, ids, settings, requestor))
+
+  private case object Tick extends NoSerializationVerificationNeeded
 }
 
 /**
  * INTERNAL API: Actor responsible for starting entities when rememberEntities is enabled
  */
 private[akka] class RememberEntityStarter(
+  region:    ActorRef,
   typeName:  String,
   shardId:   ShardRegion.ShardId,
   ids:       Set[ShardRegion.EntityId],
   settings:  ClusterShardingSettings,
-  requestor: ActorRef
-) extends Actor {
+  requestor: ActorRef) extends Actor with ActorLogging {
 
   import context.dispatcher
   import scala.concurrent.duration._
+  import RememberEntityStarter.Tick
 
-  case object Tick
-
-  val region = ClusterSharding(context.system).shardRegion(typeName)
```

Comment:
some tests are not using the extension so we can't grab the region via the extension here